### ISSUE TITLE
Plot TS vs time

### ISSUE THREAD

**Issue by kripper** [allinurl/goaccess#2160]
Nice work.

One important metric developers need to measure their web app/site performance is TS (time served) vs time.
You usually want to see when there are performance issues causing the requests to take more time to be served (for example, because the DB server is taking more time to execute the queries or some script is processing too much data).

Maybe the "TIME DISTRIBUTION" panel usged together with the --hour-spec=min argument can be used for this purpose, but adding TS times as plot metrics is missing.

It would also be very usefull to be able to examine more in detail the time periods where you have high TS spikes (like a zoom feature) and maybe be able to identify those requests responsible for this spikes (like using a profiler).

If goaccess was not designed with this in mind, can you please suggest some other tool for this "profiling" purposes?

**Comment by 0bi-w6n-K3nobi** [comment]
Hi @kripper 

Do you  took a look into real-time site at [live demo](https://rt.goaccess.io)?
Observe the columns `Avg. T.S.`, `Cum. T.S.` and `Max. T.S.`. Is it values enough for you? Or not.

For Zoom... Yeah, It would be necessary to do some development for such a filter.

**Comment by kripper** [comment]
Yes, you can select the columns you want to display and if you click on the column, the data is sorted ascending or descending and the plot shows the hours/minutes with higher values first, but you cannot see the data vs time.

What we need is very simple: plot TS vs time, ie. plot oldest values left and latests values on the right.

**Comment by 0bi-w6n-K3nobi** [comment]
I see.... I believe that understood what you want.
But... Which Panel is this coherent?

For example.... 
At `Requested Files` or `Static Requests` Panel, do you need remove `Hits` and `Visitors` columns, because it will not be more important in this context. 
And add `Time` column -- maybe with value between 1 and 10 minutes [`DT`].
Ok... until here.

Well... each request not more will be agrouped, and data will spread along data set, even that add `Time` column.
Alternatively, we can groupping request by Time, like a `Operating Systems` panel, where each group is composed by Time Interval, choiced previously into `DT`.
So... I can not see so that the chart will serve in these panels, in this context.

Can you draw the aspect/appearance of Chart and Data Set? -- For my best understanding.
